src/Core/src/Platform/Android/TransformationExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AView = Android.Views.View;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		public static void UpdateTranslationX(this AView nativeView, IView view)
+		{
+			if (nativeView.Context != null)
+				nativeView.TranslationX = nativeView.Context.ToPixels(view.TranslationX);
+		}
+
+		public static void UpdateTranslationY(this AView nativeView, IView view)
+		{
+			if (nativeView.Context != null)
+				nativeView.TranslationY = nativeView.Context.ToPixels(view.TranslationY);
+		}
+
+		public static void UpdateScale(this AView nativeView, IView view)
+		{
+			nativeView.ScaleX = (float)view.Scale * (float)view.ScaleX;
+			nativeView.ScaleY = (float)view.Scale * (float)view.ScaleY;
+		}
```

Comment:
Would this be worth splitting into Scale and ScaleX/ScaleY. Scale can just call ScaleX/ScaleY. This way we don't have to update both props each time.

Reply:
Good point. changed!